loops-and-arrays.js

```diff
@@ -21,13 +24,18 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 
 const numbers = [2, 4, 5, 29, 38];
 
+for (let i = 0; i < 5; i++){
+    console.log(numbers[i] * 3)
+}
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
 // console.log(numbers) geeft: [2, 4, 5, 29, 38];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [ 6, 12, 15, 87, 114 ];
 
```

Comment:
heb ik zo gedaan:

const numbers = [2, 4, 5, 29, 38];

console.log(numbers)
for (let i = 0; i < numbers.length; i++) {
    console.log(numbers[i]*3)
}

dus met 'numbers.length'